packages/kbn-es/src/install/install_archive.ts

```diff
@@ -20,6 +20,7 @@ import { Artifact } from '../artifact';
 import { parseSettings, SettingsFilter } from '../settings';
 import { log as defaultLog } from '../utils/log';
 import { InstallArchiveOptions } from './types';
+import { isFile, copyFileSync } from '../utils/extract_config_files';
```

Comment:
Re-using some functions from `extract_config_files.ts` that we use to move files to ES config directory, but specified as ES server args: `es_arg=path/to/file`

Reply:
__optional nit:__ it'd probably be more consistent if you export functions from `extract_config_files` in `utils/index.ts` and shorten import here, but I'll leave this to the code owners 🙂